services/motion/builtin/move_request.go

```diff
@@ -58,8 +58,9 @@ const (
 // moveRequest is a structure that contains all the information necessary for to make a move call.
 type moveRequest struct {
 	requestType requestType
-	// origin is only set if requestType == requestTypeMoveOnGlobe
-	origin            spatialmath.GeoPose
+	// geoPoseOrigin is only set if requestType == requestTypeMoveOnGlobe
+	geoPoseOrigin     spatialmath.GeoPose
+	poseOrigin        spatialmath.Pose
```

Comment:
If we have a geoPoseOrigin, then there shouldn't also be a poseOrigin, right? If that's correct can you comment saying these are mutually exclusive?

Reply:
> If we have a geoPoseOrigin, then there shouldn't also be a poseOrigin, right?

Right _now_ that is not the case. But after some thought, I think this delineation makes a lot more sense. I will update the comments. Good question!